pkgs/development/python-modules/gtimelog/default.nix

```diff
@@ -1,27 +1,43 @@
-{ lib, fetchFromGitHub, makeWrapper
-, glibcLocales, gobject-introspection, gtk3, libsoup, libsecret
-, buildPythonPackage, python
-, pygobject3, freezegun, mock
+{ lib
+, fetchFromGitHub
+, makeWrapper
+, python
+, buildPythonPackage
+, atk
+, gdk-pixbuf
+, glibcLocales
+, glib-networking
+, gobject-introspection
+, gtk3
+, harfbuzz
+, libsecret
+, libsoup
+, pango
 }:
-
 buildPythonPackage rec {
   pname = "gtimelog";
-  version = "unstable-2020-05-16";
+  version = "unstable-2022-10-28";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    rev = "80682ddbf9e0d68b8c67257289784f3b49b543d8";
-    sha256 = "0qv2kv7vc3qqlzxsisgg31cmrkkqgnmxspbj10c5fhdmwzzwi0i9";
+    rev = "73d2228cea7e500858075a139fd321b1ccbae0d1";
+    sha256 = "sha256-g6gU95uvBhBbB4b1McE2LuVPDCjdmmhBX3lVyCY96zA=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
   buildInputs = [
-    glibcLocales gobject-introspection gtk3 libsoup libsecret
+    glibcLocales
+    gobject-introspection
+    gtk3
+    libsoup
+    libsecret
   ];
 
-  propagatedBuildInputs = [
-    pygobject3 freezegun mock
+  propagatedBuildInputs = with python.pkgs; [
```

Comment:
Why this? Stuff in `pkgs/development/python-modules` should just accept Python packages as arguments.

Not that apps that are not meant to be used as Python libraries belong in this directory. But if the file is moved, it should be done in a separate commit from the changes, so that Git can still track the history.